### DISCUSSION TITLE
Potential MVTLayer issue; points seem to be pickable only in their originating tile?

### DISCUSSION THREAD

**Discussion by DanielBoa** [visgl/deck.gl#8128] [Q&A] [answered]
Not sure if this is a bug or an issue with a missing piece of layer configuration hence asking here.

I have a layer with many points, the size of which fluctuates enough for their rendered area to "bleed" out of their originating tiles. When this happens the picking API only seems to work for these points within their originating tile.

[Here's a codesandbox to illustrate what I mean](https://codesandbox.io/s/deckgl-mvt-point-pickable-issue-qhrtp9).

**Comment by Pessimistress** [comment, marked as answer]
I believe it's a bug. Can you open an issue?

**Reply by DanielBoa** [reply]
Sure 👍

**Comment by DanielBoa** [comment]
[I opened an issue to track.](https://github.com/visgl/deck.gl/issues/8134).
